### ISSUE TITLE
build warning, maybe add plugin title

### ISSUE THREAD

**Issue by cosinekitty** [epadilla/patina#1]
Hi! I saw you submitted your plugin to VCV Rack. This is a fun and original idea, so I wanted to try it out. I cloned and built it, and encountered a little build warning:

```
src/memorypad.cpp: In member function ‘virtual void MemoryPad::process(const rack::engine::Module::ProcessArgs&)’:
src/memorypad.cpp:102:26: warning: comparison of unsigned expression in ‘< 0’ is always false [-Wtype-limits]
  102 |       if (_lastPathIndex < 0) {
      |           ~~~~~~~~~~~~~~~^~~
```

It looks like this could be a problem if _currPathDirectionFwd is false and you underflow _lastPathIndex.

Also thought you might consider adding the text "Patina" to the top of your panel design. Sometimes people will use your module in a video, then post the video. Other people will see it and want to try it also. This will help the viewers of the video figure out how to find it in the VCV Rack Library.

**Comment by cosinekitty** [comment]
By the way, Memory Pad is VERY FUN!  I think people are really going to find good uses for this.  Great job.

**Comment by epadilla** [comment]
Hey, thanks for checking it out! Interesting, I didn't get that warning, but def seems like the comparison doesn't make sense. I went back to this code block and I think this check is not actually needed–I was just being overly protective, so I removed it. (https://github.com/epadilla/patina/commit/456a427c2a5c6e8bc50e8d9c4738665c2c386e74)

Thanks for the suggestion about adding the brand name. Make sense that this would help people find it.

Thank you so much for your comments!! It's very exciting to see some good responses about it.
